### PR TITLE
Regression test for #718 ("extends" recursion error)

### PR DESCRIPTION
**Pull request by yprez**

### CODE REVIEW

File: compressor/tests/test_templates/test_extends_recursion/admin/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{% extends "admin/index.html" %}
```
Comment (yprez) [inline compressor/tests/test_templates/test_extends_recursion/admin/index.html]: This is what causing it to fail, commenting it out fixes the tests.

**Comment by yprez** [comment]
Not the prettiest way to test this, but it was the best I could come up with at this point...

**Comment by diox** [comment]
This is helpful, thanks. No idea when I/someone will have the time to fix it, but having a clear failing test is a great start.

**Comment by yprez** [comment]
I tried finding the source of the bug yesterday but without much luck. I'll try taking a fresh look at it again at some point...

**Comment by yprez** [comment]
Superseded by #728

**Pull request closed without merging**